backup.sh

```diff
@@ -51,6 +51,10 @@ backup() {
 }
 
 upload_to_gcs() {
+  if [[ ! "$GCS_BUCKET" =~ gs://* ]]; then
```

Comment:
This is not necessary. It adds behaviour that certain users might not understand.
If you want to do this I would recommend creating a new var (eg: `GCS_BUCKET_NAME`) and prioritise it over `GCS_BUCKET`.
However, you need to modify the `README` so that everyone understands.
If the actual behaviour is misleading than I think it's simply better to update the `README`.